### PR TITLE
Adding skip-deploy bitops configuration option

### PR DESCRIPTION
**Pull request by PhillypHenning**

### CODE REVIEW

**Comment by LeoDiazL** [comment]
Code looks good.

File: bitops.schema.yaml [hunk 1]
```diff
@@ -24,6 +24,10 @@ terraform:
             export_env: TERRAFORM_COMMAND
             required: True
             default: plan
+          skip-deploy:
+            type: boolean
+            parameter: skip-deploy
+            export_env: TERRAFORM_SKIP_DEPLOY
```
Comment (armab) [inline bitops.schema.yaml:30]: Can we document it in the README as well?

Same is required for the official docs https://bitops.sh/tool-configuration/configuration-terraform/

File: deploy.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ export TERRAFORM_ROOT_OPERATIONS="$BITOPS_OPSREPO_ENVIRONMENT_DIR"
 export BITOPS_SCHEMA_ENV_FILE="$TERRAFORM_ROOT_OPERATIONS/ENV_FILE"
 export SCRIPTS_DIR="$TERRAFORM_ROOT_SCRIPTS/scripts"
 
-if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ]; then
+if [ "$SKIP_DEPLOY_TERRAFORM" == "true" ] || [ "$TERRAFORM_SKIP_DEPLOY" == "true" ]; then
```
Comment (armab) [inline deploy.sh:13]: What's the difference between `SKIP_DEPLOY_TERRAFORM` and `TERRAFORM_SKIP_DEPLOY`?

**Review by armab — CHANGES_REQUESTED** [review]
Looks like we're missing some documentation updates here and there:

https://github.com/bitops-plugins/terraform#terraform-bitops-schema
https://github.com/bitops-plugins/terraform#skip_deploy_terraform
https://bitops.sh/tool-configuration/configuration-terraform/

Please also update the PR title to be more clear what this PR is all about.

File: deploy.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ export TERRAFORM_ROOT_OPERATIONS="$BITOPS_OPSREPO_ENVIRONMENT_DIR"
...
   echo "SKIP_DEPLOY_TERRAFORM is set.  Skipping."
```
Comment (armab) [inline deploy.sh:14]: If we're replacing old format ENV `TERRAFORM_SKIP_DEPLOY` with a new one `TERRAFORM_SKIP_DEPLOY ` then message should be updated to a new version too: 
```suggestion
  echo "TERRAFORM_SKIP_DEPLOY is set.  Skipping."
```

**Comment by PhillypHenning** [comment]
- configuration schema added to schema (https://github.com/bitops-plugins/terraform/tree/skip-patch#skip-deploy)
- Removed references to `SKIP_DEPLOY_TERRAFORM` environment variable
- https://github.com/bitovi/bitops/pull/354

**Review by armab — APPROVED** [review]
👍

**Pull request merged**